src/features/HelpCompletion.ts

```diff
@@ -113,20 +124,20 @@ class TriggerFinder {
 }
 
 class HelpCompletionProvider {
-    private triggerFinderBlockComment: TriggerFinder;
     private triggerFinderLineComment: TriggerFinder;
     private lastChangeText: string;
     private lastChangeRange: Range;
     private lastDocument: TextDocument;
     private langClient: LanguageClient;
+    private settings: Settings.ISettings;
 
     constructor() {
-        this.triggerFinderBlockComment = new TriggerFinder("<#");
         this.triggerFinderLineComment = new TriggerFinder("##");
```

Comment:
Could this be set to the settings value rather than "##"? One less string to manage. If it's less performant we don't have to do this change.

Reply:
Oh! I see now, you're right. Somehow I was thinking you could set the trigger to Block or Line. `<#` or `##`. Ignore me :)